components/Cards/examples/CardsCollectionTintingExample.m

```diff
@@ -105,10 +129,20 @@ - (CGFloat)collectionView:(UICollectionView *)collectionView
   return 8.f;
 }
 
+#pragma mark - actions
+
+- (void)toggleModes {
+  _interactable = !_interactable;
+  NSString *buttonTitle = _interactable ? @"DEFAULT" : @"INTERACTABLE";
```

Comment:
Maybe "static" and "interactive" are better? I feel like "default" and "interactive" are not clear opposites. What do you think?